pkg/pillar/Dockerfile.in

```diff
@@ -15,12 +13,9 @@ RUN git reset --hard b41569d397d3e66099cde07d8eef36b2f42dd0ec
 COPY fscrypt/* ./
 RUN patch -p1 < patch01-no-pam.diff && \
     patch -p1 < patch02-rotate-raw-key.diff && \
-    make && make install
-
-ARG GOVER=1.15.3
```

Comment:
Where do we update the version of GO in the new scheme? I don't see it inside eve-alpine

Reply:
Currently we take the default go version that comes with alpine (see `pkg/alpine/mirrors/3.13/community`) which matches what we need. However, over time, we can make that dependency more explicit in the `pkg/alpine` Dockerfile